### ISSUE TITLE
Version 2.1

### ISSUE THREAD

**Issue by LecrisUT** [spglib/spglib#239]
Should we push some `2.1-rc` releases with the current changes? Mostly we will need to finish checking if everything is ok moving forward.
- [ ] Are the cmake options all ok in the new form?
- [ ] Naming of the targets and other conventions are ok?

Otherwise there are a few things we might want to finish before publishing `2.1.0`
- [x] #236 
- [x] #240 

Let me know of any other things to add to the list

**Comment by atztogo** [comment]
Thanks @LecrisUT. On cmake, I have no idea, but I am satisfied. I think we don't need to be so serious on the minor release.  If we feel further change after the v2.1 release, we can do it at nano versions. More Fortran tests should be written but no need to be immediate. I will do it little by little irrespective of versions.

**Comment by LecrisUT** [comment]
`nano` releases are meant to be bug-fixes, i.e. no significant changes, but if people find that there is some error in how the library runs, they might check if there is a quick fix released. I'm ok with more frequent minor releases though.

Ok, on my side the only part that I would want is #236. Because we have done a lot of overhaul, people might be looking there for how to configure and might get confused.

**Comment by atztogo** [comment]
I will work on #236.

**Comment by LecrisUT** [comment]
I think we now have a PR for all of the things we would need for this 2.1 release. And with that I think we have all of the tooling infrastructure done, except maybe for Doxygen documentation.

I haven't touched the conda-forge at all and I'm not sure how to approach it, could use some help on that.

**Comment by lan496** [comment]
@LecrisUT @atztogo 
Are there any todos for releasing v2.1?
I think we can release it now, and it is better to finish quickly because the last release (v2.0.2) was nine months ago!

**Comment by LecrisUT** [comment]
The build wheel (or at least the tests) are still broken right now. We could postpone the test fixes for later and revert to the older version of mac builder. Please poke me a bit this weekend to fix that.

**Comment by lan496** [comment]
Thank you for pointing this out. Can you link a corresponding issue later? I've gotten lost...

**Comment by LecrisUT** [comment]
These are #287 and #286

**Comment by LecrisUT** [comment]
After #287 and #315 I am ok with pushing a new release.
- First push a `rc` tag to make sure everything is in order with the CI and everything
- Then check on conda that a PR is psuhed and everything
- I will check the fedora packaging to see if everything is ok. I'll need to create a PR there manually until pre-release are available on packit
- If all are ok, can push another tag for a full release, and then bump the cmake version

**Comment by lan496** [comment]
Thanks a lot! I will create a PR to release the next version by following https://github.com/spglib/spglib/blob/develop/Contributing.md#releasing-a-new-spglib-version

**Comment by LecrisUT** [comment]
@lan496 Wait for #321

**Comment by lan496** [comment]
@LecrisUT Sure.
